doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -412,7 +412,7 @@ Explicit Universes
    | _
    | @qualid
    univ_decl ::= @%{ {* @ident } {? + } {? %| {*, @univ_constraint } {? + } } %}
-   cumul_univ_decl ::= @%{ {* {? {| = | + | * } } @ident } {? + } {? %| {*, @univ_constraint } {? + } } %}
+   cumul_univ_decl ::= @%{ {* {? {| = | + | * } } @ident } {? + } {| %| {*, @univ_constraint } {? + } %} | {| %} | %|%} } }
```

Comment:
The presentation of this production should not change. This is an implementation detail that we need to take care of the special case where you put no space between the `|` and the final `}` (case of an empty list of universe constraints).

Reply:
It's a new production that needed the same edit that `univ_decl` has, which I added.  They are very similar.  It's tempting to deem it a semantic restriction:

![image](https://user-images.githubusercontent.com/1253341/100144731-0f0f8680-2e4c-11eb-8c03-b5c559fa4433.png)